starter-code/styles/modules.css

```diff
@@ -66,3 +72,27 @@
          - The main-nav should be all on one row, with padding spaced better
            for a larger screen.
 */
+
```

Comment:
Above this, in Line 70, is the instruction to hide the Hamburger, which is not happening in your code. That needs to be fixed!  I'll give you a hint:  there is a ::before somewhere that you do not need, and once eliminating it, fixes the problem. Happy hunting!